database/scrapers/departments.ts

```diff
@@ -69,45 +67,28 @@ export default class DepartmentsScraper extends BaseScraperModule {
       (await Promise.all(
         this.directusSchemas.map((schema) =>
           this.semaphore.runTask(() =>
-            this.fetchJSON(
+            this.fetchDirectusJSON(
               `https://admin.topwr.solvro.pl/items/${schema}?limit=-1`,
               schema,
-            ),
+            ).addErrorContext(`Failed for schema ${schema}`),
```

Comment:
same as above: the string should make sense as an error message - even if the context stack would reveal this, do mention *what* it actually failed to do (i.e. failed to download the schema)
also if the error context is generated at runtime (for example if it includes a template string) then it is preferred to wrap it in an arrow function, which will only be evaluated on error.

though as the fetch methods already include info about which schema failed (probably cuz they were made before i "borrowed" error handling schemes from rust) you may as well remove the error context, as it's redundant.